src/Microsoft.ML.Data/Training/TrainerUtils.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Generic;
 using Microsoft.ML.Core.Data;
 using Microsoft.ML.Data;
+using Microsoft.ML.EntryPoints;
```

Comment:
This doesn't look right.

Reply:
right! I'll move Optional in Data. 

---
In reply to: [248115282](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248115282) [](ancestors = 248115282)

Reply:
It just strange what we have EntryPoints as dependency. But I guess it's related to fact what Arguments class shared with entrypoints.

---
In reply to: [248152842](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248152842) [](ancestors = 248152842,248115282)